Prose/ProseLib/Tests/ProseCoreTCATests/Helpers/BareJid+Tests.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+import ProseCoreClientFFI
+
+extension BareJid: ExpressibleByStringLiteral {
+    public init(stringLiteral value: String) {
+        self = try! ProseCoreClientFFI.parseJid(jid: value)
```

Comment:
`try!` is OK here, but (I think) a `fatalError` with a message would be better 💁🏻‍♂️